crates/merod/src/cli/run.rs

```diff
@@ -13,8 +13,34 @@ use crate::cli::RootArgs;
 /// Run a node
 #[derive(Debug, Parser)]
 pub struct RunCommand {
+    /// Enable authentication (deprecated, use --auth-mode instead)
     #[arg(long, default_value_t)]
+    #[clap(hide = true)]
```

Comment:
are you trying to replicate something like this:

https://github.com/calimero-network/core/blob/fa5ff2f8f0d0e0e23081d0f45a419ea2ab668062/crates/merod/src/cli/init.rs#L103-L109